doc/manual/manual.tex

```diff
@@ -320,12 +320,26 @@ \subsection{Referencing \aspect{}}
 }
 \end{lstlisting}
 
-
+Cite the  most recent release of ASPECT (v1.5.0) as:
+\begin{lstlisting}[frame=single,language=tex]
```

Comment:
while I agree, there is one problem with this:
a specific release (for example 2.0) can not contain the citation information for that release (because we don't have the DOI when we package the release). This means it will always be out of date. :-(
My suggestion would be to remove all info on how to cite here and always link to the same place.